src/otx/core/model/module/base.py

```diff
@@ -203,7 +203,7 @@ def load_state_dict(self, ckpt: dict[str, Any], *args, **kwargs) -> None:
     @property
     def lr_scheduler_monitor_key(self) -> str:
         """Metric name that the learning rate scheduler monitor."""
-        return "val/loss"
+        return self.scheduler[-1].monitor
```

Comment:
scheduler[0] is might `WarmScheduler`, so need to check the `monitor` key of last scheduler.

Reply:
There is no guarantee that `self.scheduler[-1]` has `monitor` attribute and there seems no use of this property after https://github.com/openvinotoolkit/training_extensions/pull/2910. Please just remove this property.